test_config.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
```

Comment:
Can we put this test in the `tests.flytekit.unit.configuration` package and also make it assert something. Just for good measure.